ignite/engine/engine.py

```diff
@@ -290,39 +269,7 @@ def execute_something():
             See :class:`~ignite.engine.events.Events` for more details.
 
         """
-        if isinstance(event_name, EventsList):
-            for e in event_name:
-                self.add_event_handler(e, handler, *args, **kwargs)
-            return RemovableEventHandle(event_name, handler, self)
-        if (
-            isinstance(event_name, CallableEventWithFilter)
-            and event_name.filter != CallableEventWithFilter.default_event_filter
-        ):
-            event_filter = event_name.filter
-            handler = self._handler_wrapper(handler, event_name, event_filter)
-
-        self._assert_allowed_event(event_name)
-
-        event_args = (Exception(),) if event_name == Events.EXCEPTION_RAISED else ()
-        try:
-            _check_signature(handler, "handler", self, *(event_args + args), **kwargs)
-            self._event_handlers[event_name].append((handler, (self,) + args, kwargs))
-        except ValueError:
-            _check_signature(handler, "handler", *(event_args + args), **kwargs)
-            self._event_handlers[event_name].append((handler, args, kwargs))
-        self.logger.debug(f"added handler for event {event_name}")
-
-        return RemovableEventHandle(event_name, handler, self)
-
-    @staticmethod
-    def _assert_non_filtered_event(event_name: Any) -> None:
-        if (
-            isinstance(event_name, CallableEventWithFilter)
-            and event_name.filter != CallableEventWithFilter.default_event_filter
-        ):
-            raise TypeError(
-                "Argument event_name should not be a filtered event, " "please use event without any event filtering"
-            )
+        return super(Engine, self).add_event_handler(event_name, handler, *args, **kwargs)
```

Comment:
@KickItLikeShika you resolved the comment, yet you didn’t answer.

Reply:
Very sorry about that, but I answered your question here https://github.com/pytorch/ignite/pull/2107#discussion_r669973702, I thought you saw it and agreed so I resolved the comment, again sorry.
I think keeping the full docs in the Engine docs and minimized docs in the base class if something nice, what do you think?